build/helm.yaml

```diff
@@ -31,4 +31,23 @@ roleRef:
 subjects:
   - kind: ServiceAccount
     name: helm
-    namespace: kube-system
\ No newline at end of file
+    namespace: kube-system
+---
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Curious - what is this for?

Or at least making sure this was deliberate :smiley: